[2021-02-23 14:59:12,853] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_trigger_target_dag.run_this 2021-02-23T14:57:17.450328+00:00 [queued]>
[2021-02-23 14:59:13,366] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_trigger_target_dag.run_this 2021-02-23T14:57:17.450328+00:00 [queued]>
[2021-02-23 14:59:13,393] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:59:13,399] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-23 14:59:13,427] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:59:13,658] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): run_this> on 2021-02-23T14:57:17.450328+00:00
[2021-02-23 14:59:13,698] {standard_task_runner.py:52} INFO - Started process 295 to run task
[2021-02-23 14:59:13,803] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_trigger_target_dag', 'run_this', '2021-02-23T14:57:17.450328+00:00', '--job-id', '111', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py', '--cfg-path', '/tmp/tmpwrhhrtbl', '--error-file', '/tmp/tmpvo3oj2cl']
[2021-02-23 14:59:13,815] {standard_task_runner.py:77} INFO - Job 111: Subtask run_this
[2021-02-23 14:59:14,214] {logging_mixin.py:104} INFO - Running <TaskInstance: example_trigger_target_dag.run_this 2021-02-23T14:57:17.450328+00:00 [running]> on host 0513e3a4eab6
[2021-02-23 14:59:15,062] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_trigger_target_dag
AIRFLOW_CTX_TASK_ID=run_this
AIRFLOW_CTX_EXECUTION_DATE=2021-02-23T14:57:17.450328+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-02-23T14:57:16.555874+00:00
[2021-02-23 14:59:15,104] {logging_mixin.py:104} INFO - Remotely received value of Hello World for key=message
[2021-02-23 14:59:15,109] {python.py:118} INFO - Done. Returned value was: None
[2021-02-23 14:59:15,319] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_trigger_target_dag, task_id=run_this, execution_date=20210223T145717, start_date=20210223T145912, end_date=20210223T145915
[2021-02-23 14:59:17,049] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-02-23 14:59:17,711] {local_task_job.py:146} INFO - Task exited with return code 0
